### ISSUE TITLE
is something wrong toggle & draggable？

### ISSUE THREAD

**Issue by moooyy** [quasarframework/quasar#1806]
A dom contains <q-toggle> component, and add draggable="true" to its parentNode, like this:
```html
<div draggable="true">
    <q-toggle />
</div>
<div draggable="true">
    <q-toggle />
</div>
<div draggable="true">
    <q-toggle />
</div>
```
When you dragging the first element, the others will follow with mouse,  looks like 'draggable' cover all doms, maybe unmatched label cause this

Have anyone ever met this situation？

**Comment by pdanpdan** [comment]
This is the exact code you're using? Can you try closing the double quote after `true` in all cases and try again?

**Comment by rstoenescu** [comment]
Hi, Should be fixed in v0.17. Please reopen if you still experience it.
